### ISSUE TITLE
UPRN type code in Property model

### ISSUE THREAD

**Issue by jdziek** [reapit/foundations#8823]
**Is your feature request related to a problem? Please describe.**
The majority of the industry uses UPRN code, or equivalent, to verify and store address information.  Without those, it is difficult to verify the validity of an address when on a programmatical level. Fuzzy type search/filter does not provide results good enough to be reliable when dealing with property transactions.

**Describe the solution you'd like**
Implementing UPRN type code (UPRN, UDPRN, UMRRN  etc) would make it easier to validate and synchronise the addresses between the apps and integrations. Having one of those in the property model would make integrations much more reliable.

**Additional context**
Basically, when we get a webhook from  Reapit with an address, we need to perform a fuzzy search with another API to filter out a valid address with UPRN.  This method is not reliable and I wouldn't trust it to work in a lot of cases, which simply is not good enough for our clients. The UPRN type codes are used across most of the industry at this point and a lack of them may result in integration issues.
